.github/workflows/preview-env-deploy.yml

```diff
@@ -73,21 +66,14 @@ jobs:
           env: ${{ github.event.repository.name }}
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Find build status comment
-        uses: peter-evans/find-comment@v3
-        id: find-build-comment
-        with:
-          issue-number: ${{ github.event.number }}
-          body-includes: <!-- preview-env -->
-
       - name: Update comment with upload status
-        uses: peter-evans/create-or-update-comment@v4
+        uses: mshick/add-pr-comment@v2
         with:
-          comment-id: ${{ steps.find-build-comment.outputs.comment-id }}
-          issue-number: ${{ github.event.number }}
-          body: |
-            :arrow_up: The preview environment build results for commit ${{ github.sha }} is being uploaded. This usually takes 3-4 minutes. <!-- preview-env -->
-          edit-mode: replace
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message-id: preview-env-status
```

Comment:
`message-id` is also not necessary, if you don't plan on having more than one comment you want to update throughout the process. There's a default value like `add-pr-comment`.